activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+*   Add `assert_differences` to `ActiveSupport::Testing::Assertions`
```

Comment:
Oh, so it is fine. I'll change all of them when reviewing the CHANGELOG in the release